test/quantization/test_quantized_module.py

```diff
@@ -864,3 +950,5 @@ def test_cell_api(self):
                 result_module = cell_dq(x, state[rnn_type])
                 self.assertEqual(result[0], result_module[0], msg="RNNCell module API failed")
                 self.assertEqual(result[1], result_module[1], msg="RNNCell module API failed")
+                check_weight_bias_api(cell_dq)
+                check_serialization(cell_dq, cell_dict[rnn_type](**kwargs))
```

Comment:
Can we factor out `check_weight_bias_api` and `check_serialization` into common_quantization since it is being used in multiple places?